src/HornedBeast.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './App.css';
+
+class HornedBeast extends React.Component {
+  render () {
+    return(
+      <>
+        <h2>
```

Comment:
Think about bootstrap. Is there something you could use in there to organize this data a little better?